vital/api/sleep.py

```diff
@@ -1,7 +1,13 @@
+import logging
 from typing import List, Mapping, Optional
 
+import arrow
+
 from vital.api.api import API
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
```

Comment:
How about `import warnings`? always shows up in the logs and less annoying with which root logger is being used etc